ReactAndroid/src/main/java/com/facebook/react/Experiments.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.react;
+
+import java.util.HashSet;
+import java.util.Set;
+
+
+/**
+ * This class exposes a way to enable/disable exprimental features in React Native.
+ */
+
+public class Experiments {
+
+  // Enables Hot Module Replacement menu item in the developer menu
+  public static String HOT_MODULE_REPLACEMENT = "HOT_MODULE_REPLACEMENT";
+
+  private static Set<String> enabledExperiments;
+
+  /**
+   * Enables an experiment
+   *
+   * @param experiment - the experiment to enable
+   */
+  public static void enableExperiment(String experiment) {
+    if (enabledExperiments == null) {
+      enabledExperiments = new HashSet<>();
+    }
+    enabledExperiments.add(experiment);
+  }
+
+  /**
+   * Disbales an expriment
+   *
+   * @param experiment - the experiment to disable
+   */
+  public static void disableExperiment(String experiment) {
+    if (enabledExperiments != null) {
+      enabledExperiments.remove(experiment);
+    }
+  }
+
+  /**
+   * Checks if an expriment is enabled
+   *
+   * @param experiment - the experiment to check for
+   */
```

Comment:
It's a matter of personal preference but these three methods have so clear names they probably don't require Javadoc. If you prefer keep it but please remove the `@param experiment` - these don't add meaning to what's already in the code.